backend/app/api/endpoints/kind/common.py

```
@@ -0,0 +1,286 @@
+        HTTPException: When user does not exist
+    """
+    try:
+        if db:
+            user_service.get_user_by_id(db, user_id)
+        else:
+            user_service.get_user_by_id(None, user_id)
+    except NotFoundException as e:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=str(e)
+        )
```

Ruff (0.14.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)